frontend/src/components/games/list/GameTable.tsx

```diff
@@ -18,115 +18,110 @@ import {
     RenderCohort,
     RenderOwner,
     RenderPlayersCell,
-    RenderResult,
+    RenderTimeControl,
 } from './GameListItem';
 
 export const gameTableColumns: GridColDef<GameInfo>[] = [
     {
         field: 'cohort',
         headerName: 'Cohort',
-        width: 65,
         renderCell: (params: GridRenderCellParams<GameInfo, string>) =>
             RenderCohort(params.row),
     },
-    {
-        field: 'owner',
-        headerName: 'Uploaded By',
-        minWidth: 150,
-        renderCell: (params: GridRenderCellParams<GameInfo, string>) =>
-            RenderOwner(params.row),
-    },
     {
         field: 'players',
         headerName: 'Players',
         valueGetter: (_value, row) =>
             `${row.headers.White} (${row.headers.WhiteElo}) - ${row.headers.Black} (${row.headers.BlackElo})`,
         renderCell: RenderPlayersCell,
-        flex: 1,
-        minWidth: 150,
+        minWidth: 200,
+        flex: 2,
     },
     {
-        field: 'result',
```

Comment:
One side effect of removing this column is that we lose the ability to sort by result, which is a rather big loss

Reply:
Was added back in.